sdk/python/featurestore_sample/notebooks/sdk_only/2. Experiment and train models using features.ipynb

```diff
@@ -647,7 +647,7 @@
      "outputs_hidden": false,
      "source_hidden": false
     },
-    "name": "gen-training-data-locally",
+    "name": "load-obs-data",
```

Comment:
This change WILL break a build of our docs.  We have an article that is using the code in this cell, referenced by the name "gen-training-data-locally".  Furthermore, we have a publishing freeze until Nov 15, so cannot change the reference until then.

Reply:
The cell name appears later in the notebook so we're ok.